src/test/java/seedu/address/logic/command/GoToCommandTest.java

```diff
@@ -12,7 +12,7 @@
     @Test
     public void execute_goto_success() {
         try {
-            Index index = TypicalIndexes.INDEX_FIRST_PERSON;
+            Index index = TypicalIndexes.INDEX_FIRST;
```

Comment:
This one I made quite some changes. So when you resolving conflict can just use mine.